README.md

```diff
@@ -1,6 +1,6 @@
 # Monero
 
-Copyright (c) 2014-2020 The Monero Project.   
+Copyright (c) 2014-2020, The Monero Project.   
```

Comment:
Still 2020

Reply:
We have another PR for the readme year open.

Reply:
I've just read it, sorry.
Wouldn't it be better not to change the main Readme at all (mind the colon) for now to avoid the conflict?

Reply:
Also, I still think, that retriggering the checks would increase the credibility, just in case.

Reply:
“ Wouldn't it be better not to change the main Readme at all (mind the colon) for now to avoid the conflict?”

Not only did I update copyrights, I also made them all uniform. Some said ‘Copyright (c) yyyy The Monero Project’ and some said ‘Copyright (c), The Monero Project’. Now all copyrights are uniform and finding them is slightly easier.

Reply:
This is great, of course. But if you're excluding the main Readme for a valid reason, it should really be completely excluded, not partially excluded. Alternatively, the PR should overwrite the other PR's change completely (in this case 2020->2021, together with the comma).

Reply:
I gave a wrong recipe for retriggering checks. It should have been:

```
git commit -m "retrigger checks" --allow-empty
git push
```
Then
```
git rebase -i master
git push -f
```